install

```diff
@@ -49,18 +55,24 @@ if [ $dont_skip_types = 1 ]; then
     python -m crds.tobs
 fi
 
-pip install .
+if [ $dev_install = 0 ]; then
+    echo "Doing setup install..."
+    pip install .
```

Comment:
Where does the pip install occur when `$dev_install = 1`?

Reply:
This was the main goal. If `crds` had been originally installed with `pip -e`, there is no desire to have things installed again.

That being said, and some enlightenment from the above comment, what may really be needed is another script that does just the checksum update (and whatever other operation), and restore `install` back to what it was, except for calling the new script.